### PR TITLE
Feature/productions schema update

### PR DESCRIPTION
**Pull request by JElgar**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -125,8 +167,13 @@ type ProductionNode implements Node {
   coverImage: GrapheneImageFieldNode
   ageRating: Int
   facebookEvent: String
+  warnings(offset: Int, before: String, after: String, first: Int, last: Int): WarningNodeConnection!
```
Comment (atoff) [inline schema.graphql:170]: Can we not node use edges here - its very overkill

File: schema.graphql [hunk 2]
```diff
@@ -125,8 +167,13 @@ type ProductionNode implements Node {
...
   slug: String!
+  cast(offset: Int, before: String, after: String, first: Int, last: Int): CastMemberNodeConnection!
+  crew(offset: Int, before: String, after: String, first: Int, last: Int): CrewMemberNodeConnection!
```
Comment (atoff) [inline schema.graphql:173]: Tempted to say same as above for cast crew and prod team - it's not worth paginating them because of how we display

**Review by JElgar — COMMENTED** [review]
LGTM

**Comment by streth11** [comment]
love it

**Review by streth11 — APPROVED** [review]
great stuff

**Pull request merged**